### ISSUE TITLE
Flexible logger configuration

### ISSUE THREAD

**Issue by alexeymarunin** [moleculerjs/moleculer#569]
I like Moleculer. Simple, powerful and flexible. And I don't like write much code - I prefer _configure_ app. For example, look at [transporter config](https://moleculer.services/docs/0.13/networking.html#Transporters) - it's a pure delight. Or [caching config](https://moleculer.services/docs/0.13/caching.html). Or [serializer](https://moleculer.services/docs/0.13/networking.html#Serialization). I can use [config](https://github.com/lorenwest/node-config) package to include YAML/JSON file with broker settings and control all my services in one place. It's really cool.

```js
// moleculer.config.js
const config = require('config')
const cfg = config.util.toObject() // now it's plain object - broker settings

// Extend/transform settings here
// ...

module.exports = cfg
```

But there is the one thing which makes me sad - it's a [logger config](https://moleculer.services/docs/0.13/logging.html). Some logging options are in `broker` section, some - in `logger` section. And I have to code logger:

```js
const Logger = require('pino')
const logger = Logger({level: 'info', prettyPrint: true})
_.extend(cfg, {
  logger: bindings => logger.child(bindings, true),
})
```

I have about 15-20 source repos based on Moleculer, and I have to write this code again and again. But if I want to replace logger? It will require ton of routine work. 

I think logger's config should be more flexible (more configurable): 

```js
const broker = new ServiceBroker({ 
    logger: 'pino' // or 'bunyan', 'winston', etc
})
```

or 

```js
const broker = new ServiceBroker({ 
    logger: {
      type: 'pino',
      options: {
        level: 'info',
        // ...
      }
})
```

And default (console) logger:
```js
const broker = new ServiceBroker({ 
    logger: {
      type: 'console',
      options: {
        level: 'info',
        colors: true,
        formatter: (o) => console.log('%j', o),
        // ...
      }
})
```

Compare with transporter or serializer config - it's look the same! Now I can configure different Moleculer's components similar way. Yes, in this way I have to install corresponding package, but it's not big problem, doesn't?

What do you think of that feature?

**Comment by icebob** [comment]
Good idea! But backward compatibility should be solved, as well.

**Comment by icebob** [comment]
I'm working on it.As it's a breaking change, it will be released in 0.14.
I'm adding built-in loggers as:
- console
- pino
- bunyan
- winston
- debug
- log4js
- file

And you can define multiple loggers with different levels, as well.

**Comment by icebob** [comment]
Implemented in #591
